### ISSUE TITLE
can UMI count matrix be input by somehow transforming ?

### ISSUE THREAD

**Issue by JianmeiZhong** [NabaviLab/SigEMD#2]
hi,
  Thanks for your great work.:)
  I read that log2(TPM+1) data are needed but not UMI gene-barcode matrix from cellranger pipeline of 10x genomics.can UMI count matrix be input by somehow transforming ?

Best,

**Comment by tianyu-github** [comment]
Yes UMI count matrix can be used, but normalization is not considered in this method so you need preprocessed the data.

**Comment by NTNguyen13** [comment]
Hi, I recently know about sigEMD, and I also used scRNA-seq from 10X pipeline.

Please correct me if I'm wrong, I understand that I can use the UMI count matrix directly, before normalization, scaling, etc., as input for sigEMD. Am I right?

**Comment by tianyu-github** [comment]
> Hi, I recently know about sigEMD, and I also used scRNA-seq from 10X pipeline.
> 
> Please correct me if I'm wrong, I understand that I can use the UMI count matrix directly, before normalization, scaling, etc., as input for sigEMD. Am I right?

No, it can not be used directly, the count matrix should be preprocessed, such as normalization, before using SigEMD.

**Comment by NTNguyen13** [comment]
Yes, thank you for clarification
